docs/content/dagster-cloud/insights/asset-metadata.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Using asset metadata with Dagster Cloud Insights | Dagster Docs"
+description: "Integrating external metrics with Dagster Insights."
+
+platform_type: "cloud"
+---
+
+# Using asset metadata with Dagster Cloud Insights (Experimental)
+
+<Note>
+  This feature is considered <strong>experimental</strong>.
+</Note>
+
+Any numeric [asset materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata) attached to your assets can be viewed and aggregated in the Dagster Insights UI. This is a good way to track usage, cost, data size, or other characteristics of your data assets over time, and across different parts of your platform.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- A Dagster Cloud account on the Enterprise plan
+- Access to the [Dagster Cloud Insights feature](/dagster-cloud/insights)
+
+---
+
+## Step 1: Emit numeric values alongside your asset materializations
+
+In order to populate Insights with materialization metadata, you must first emit numeric metadata to be aggregated.
+
+```python file=/concepts/assets/asset_materialization_metadata_none.py
+from dagster import Output, asset
+
+
+@asset
+def table1() -> Output[None]:
+    ...  # write out some data to table1
+    return Output(None, metadata={"num_rows": 25})
+```
+
+For more examples, see the [asset materialization metadata documentation](/concepts/assets/software-defined-assets#recording-materialization-metadata).
+
+## Step 2: Enable viewing numeric metadata in Dagster Cloud Insights
+
+Once one or more assets are emitting numeric metadata values, you are able to enable viewing them in the Insights UI. Note that it may take up to 24 hours for new metadata types to be reflected in the UI.
```

Comment:
```suggestion
Once one or more assets are emitting numeric metadata values, you'll be able to enable viewing them in the Insights UI. **Note**: It may take up to 24 hours for new metadata types to be reflected in the UI.
```